packages/server/test/integration/cypress_spec.js

```diff
@@ -1024,7 +1024,6 @@ describe('lib/cypress', () => {
               browser: 'electron',
               foo: 'bar',
               onNewWindow: sinon.match.func,
-              writeVideoFrame: sinon.match.func,
```

Comment:
Removed this unit assertion since it's complex to mock now and is covered by other Electron tests that record video.